be/src/cache/datacache.cpp

```diff
@@ -78,6 +87,22 @@ void DataCache::destroy() {
     LOG(INFO) << "datacache shutdown successfully";
 }
 
+bool DataCache::adjust_capacity(int64_t delta, size_t min_capacity) {
+    if (_lru_cache != nullptr) {
```

Comment:
Use `adjust_mem_capacity` to avoid confusion with disk adjustments？

Reply:
Fixed